k8s/src/main/scala/io/buoyant/k8s/istio/IdentifierPreconditions.scala

```diff
@@ -1,10 +1,20 @@
 package io.buoyant.k8s.istio
 
 import com.twitter.finagle.Path
-import istio.proxy.v1.config.{MatchCondition, RouteRule, StringMatch}
+import istio.proxy.v1.config.{HTTPRewrite, MatchCondition, RouteRule, StringMatch}
 import istio.proxy.v1.config.StringMatch.OneofMatchType
 
 trait IdentifierPreconditions {
+  case class IstioRequestMeta(
+    uri: String,
+    scheme: String,
+    method: String,
+    authority: String,
+    getHeader: (String) => Option[String]
+  )
+
+  def pathFromUri(uri: String): String = "/" + uri.split("/").drop(1).mkString("/")
```

Comment:
Actually, I don't think this is needed anymore. Originally I thought I'd need to separate out "/my/path" from "foo.bar.com/my/path".